src/main/java/com/agnieszkapawska/flashcards/domain/dtos/FlashcardDto.java

```diff
@@ -0,0 +1,18 @@
+package com.agnieszkapawska.flashcards.domain.dtos;
+
+import com.agnieszkapawska.flashcards.domain.models.QuestionTags;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlashcardDto {
+    private String question;
+    private String answer;
+    private String exampleUsage;
+    private String explanation;
+    private Set<QuestionTags> questionTagsList;
```

Comment:
```suggestion
    private Set<QuestionTags> tagsList;
```
I think it's more like flashcardTagsList but IMO tagsList is enough. We are in this class and we know that those are properties of this class. We don't need to write it in property name.